tests/unit/htdocs/class/LostpassSecurityTest.php

```
@@ -0,0 +1,443 @@
+        $this->assertInstanceOf(LostpassSecurity::class, $sec);
+    }
+
+    /* ========================================================
+     * storePayload (actkey path — column fits)
+     * ====================================================== */
+
+    #[Test]
+    public function storePayloadUsesActkeyWhenColumnFits(): void
+    {
+        // Mock DB to report actkey column as VARCHAR(100)
+        $db = $this->createMockDatabase();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)